app/cli/cmd/auth_login.go

```diff
@@ -71,6 +71,7 @@ func interactiveAuth() error {
 	// Append local callback URL
 	q := serverLoginURL.Query()
 	q.Set("callback", callbackURL.String())
+	q.Set("long-lived", "true")
```

Comment:
It would be perfect if we can use queryParamLongLived here and in the service code.

Reply:
Right, I thought about it, but there is some refactoring that would need to be done. Such as extracting those constants outside of the `internal` package in the controlplane to a shared `parent` module.

Let me revisit it. Thanks

Reply:
Updated